js/store.js

```diff
@@ -1,19 +1,34 @@
 const store = {
+  _websites: null,
+
   init() {
-    this.addListeners();
+    this._read();
+  },
+
+  async _read() {
+    const data = await browser.storage.local.get('websites');
+    this._websites = clone(data.websites);
```

Comment:
I was a bit confused about to put this piece here or in `init`. So will move this into `init`..